src/platform/forms-system/src/js/components/ErrorableCheckbox.jsx

```diff
@@ -46,6 +46,7 @@ class ErrorableCheckbox extends React.Component {
     return (
       <div className={className}>
         <input
+          aria-labelledby={this.props.ariaLabelledBy}
```

Comment:
I think this is a good change, but this ErrorableCheckbox component is duplicate code that we should get rid of eventually. I think you want to update the ErrorableCheckbox component in formation-react, and then have the PreferenceOption component use that.

Reply:
Fixed -- now importing ErrorableCheckbox from formation-react instead.

I'll open a new ticket to remove platform's ErrorableCheckbox from vets-website.